docs/review/review.yaml

```diff
@@ -0,0 +1,299 @@
+paths:
+  /reviews:
+    get:
+      security:
+        - bearerAuth: []
+      tags:
+        - Reviews
+      summary: Find all reviews
+      description: Finds and returns an array with a list of reviews.
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/definitions/reviews'
+              example:
+                - _id: 6255bc080a75adf9223df100
+                  comment: Not bad
+                  rating: 4
+                  tutor: 6255bc080a75adf9223df211
+                  student: 6255bc080a75adf9223df212
+                  createdAt: 2022-10-18T13:25:36.292Z
+                  updatedAt: 2022-10-18T13:25:36.292Z
+                - _id: 8755bc080a00adr9243df104
+                  comment: Best tutor!
+                  rating: 5
+                  tutor: 6255bc080a75adf9223df111
+                  student: 6255bc080a75adf9223df112
+                  createdAt: 2022-10-18T13:25:36.292Z
+                  updatedAt: 2022-10-18T13:25:36.292Z
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 401
+                code: 'UNAUTHORIZED'
+                message: 'The requested URL requires user authorization.'
+    post:
+      security:
+        - bearerAuth: []
+      tags:
+        - Reviews
+      summary: Create new review.
+      description: Saves a new review the DB.
+      produces:
+        - application/json
+      requestBody:
+        required: true
+        description: Create new review.
+        content:
+          application/json:
+            schema:
+              $ref: '#/definitions/review'
+            example:
+              comment: Best tutor!
+              rating: 5
+              tutor: 6255bc080a75adf9223df111
```

Comment:
to leave a review all what you need is
comment
rating
tutor & student will be taken from request
createdAt & updatedAt are not need to create the review, they will be in response only, as well  as tutor & student

Reply:
fixed